src/style.css

```
@@ -1,3 +1,63 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 500;
+  line-height: 14px;
+  font-style: normal;
+
+  --main-color: #00acdc;
```

```suggestion
  --color-blue: #00acdc;
```
Blue color not the main as for me, It's better to use more descriptive names for variables